tools/test-js-with-node

```diff
@@ -8,9 +7,11 @@ export PATH=$PATH:node_modules/.bin
 
 INDEX_JS=frontend_tests/zjsunit/index.js
 NODEJS=$(which nodejs || which node)
+
 if [ "$1" = "cover" ]; then
   # Run a coverage test with Istanbul.
-  istanbul cover "$INDEX_JS" && echo "Test(s) passed. SUCCESS!" || (echo "FAIL - Test(s) failed"; exit 1)
+  istanbul cover "$INDEX_JS";
+  ret=$?;
 
 elif [ "$1" = "-h" -o "$1" = "--help" ]; then
   echo "Usage:
```

Comment:
Not that it really matters, but did you test what the exit code will be for `--help`?